user_guide_src/source/libraries/caching.rst

```diff
@@ -155,7 +155,7 @@ Class Reference
 
         .. literalinclude:: caching/004.php
 
-        .. note:: The ``$raw`` parameter is only utilized by Memcache,
+        .. note:: The ``$raw`` parameter is only utilized by Memcached,
```

Comment:
There is no `$raw` parameter.
See https://github.com/codeigniter4/CodeIgniter4/blob/8b0c99bc8efbd7d50c545cb529e84602ba1ea6b7/system/Cache/Handlers/MemcachedHandler.php#L160

Reply:
@kenjis Does that mean we should remove this note completely? Or possibly remove this **$raw** parameter because I couldn't see it in other handlers either.

Reply:
It seems ``$raw`` was not implemented or removed.
So please remove the parameter and the note.